report_thesis/src/sections/problem_definition.tex

```diff
@@ -0,0 +1,54 @@
+Some of these are caused by \textit{matrix effects}\cite{andersonImprovedAccuracyQuantitative2017}, which is a catch-all term for any effect that can cause the intensity of emission lines from an element to vary, independent of that element's concentration. So it's unknown variables that affect the results.
+Furthermore, due to the high cost of data collection, datasets are often small, which further complicates the task of building accurate and robust models.
+
+Based on the limitations with the current MOC pipeline, as reported in \cite{p9_paper}, we identified three key areas for further investigation: dimensionality reduction, model selection, and outlier removal.
+
+In this work, we focus on dimensionality reduction and model selection over outlier removal.
+This is justified by the low incidence of outliers in the ChemCam \gls{libs} calibration dataset, as reported in \cite{p9_paper}.
+Dimensionality reduction is crucial for managing the high-dimensional nature of \gls{libs} data. % TODO: There are lots of related works which explore DR in LIBS data. We can back this up with citations.
+Furthermore, model selection shows promise in addressing the limitations of the current MOC pipeline, as it allows for the exploration of a wider range of algorithms, potentially leading to improved performance.
+We showed that advanced ensemble methods, such as \gls{gbr}, and deep learning models, such as \gls{ann}, have the potential to outperform the current MOC pipeline.
+Methods are selected based on their promise in handling high-dimensional, non-linear data. Ideally, the selected methods should also be feasible for small datasets, a common scenario in \gls{libs} analyses. 
+In order to address the aforementioned challenges, we propose to explore advanced ensemble methods and deep learning models, which have shown promise in handling high-dimensional, non-linear data.
+
+It is necessary to establish metrics to evaluate the performance of the models.
+In \cite{p9_paper}, we proposed to use the \gls{rmse} as a proxy for accuracy.
+
+\gls{rmse} is given by:
+
+\begin{equation}
+    RMSE = \sqrt{\frac{1}{n} \sum_{i=1}^{n} (y_i - \hat{y}_i)^2}
+\end{equation}
+
+where $y_i$ represents the actual values, $\hat{y}_i$ the predicted values, and $n$ the number of observations.
+
+To address robustness, we propose considering the standard deviation of prediction errors across each oxide and test instance, defined as:
+
+\begin{equation}
+    \sigma_{error} = \sqrt{\frac{1}{n-1} \sum_{i=1}^{n} (e_i - \bar{e})^2}
+\end{equation}
+
+where $e_i = y_i - \hat{y}_i$ and $\bar{e}$ is the mean error.
+
+In order to narrow down the scope of our research, we set the following constraints:
+\begin{itemize}
+    \item Prioritize normalization across individual spectrometers' wavelength ranges over full-spectrum normalization.
```

Comment:
Should we mention why we do this? And maybe reference SuperCam? The reasoning behind this constraint isn't trivially obvious to someone in the CS field.

Reply:
For style reasons, I prefer the bullets being short points that I expand upon after. This is rather common.

Reply:
Added clarification